data/strings/strings.txt

```diff
@@ -4774,6 +4774,48 @@
     zh-Hans = 3D 建筑
     zh-Hant = 3D 建築
 
+  [pref_map_3d_buildings_disabled_summary]
+    comment = Settings «Map» category: «3D buildings» summary when 3D buildings are disabled because of power saving mode
+    tags = android,ios
+    en = 3D buildings are turned off in power saving mode
+    ar = يتم إيقاف تشغيل المباني ثلاثية الأبعاد في وضع توفير الطاقة
+    be = У рэжыме энергазберажэння 3D-будынкі выключаны
+    bg = 3D сградите са изключени в режим на пестене на енергия
+    ca = Els edificis en 3D es desactiven en mode d'estalvi d'energia
+    cs = 3D budovy jsou vypnuty v úsporném režimu
+    da = 3D-bygninger er slukket i strømbesparende tilstand
+    de = 3D-Gebäude werden im Energiesparmodus ausgeschaltet
+    el = Τα τρισδιάστατα κτίρια απενεργοποιούνται σε λειτουργία εξοικονόμησης ενέργειας
+    es = Los edificios 3D se apagan en el modo de ahorro de energía
+    et = 3D-hooned lülitatakse energiasäästurežiimis välja
+    eu = 3D eraikinak itzalita daude energia aurrezteko moduan
+    fa = ساختمان های سه بعدی در حالت صرفه جویی در مصرف برق خاموش می شوند
+    fi = 3D-rakennukset sammutetaan virransäästötilassa
+    fr = Les bâtiments 3D sont désactivés en mode d'économie d'énergie
+    he = מבנים תלת מימדיים כבויים במצב חיסכון בחשמל
+    hu = A 3D épületek energiatakarékos módban ki vannak kapcsolva
+    id = Bangunan 3D dimatikan dalam mode hemat daya
+    it = Gli edifici 3D sono disattivati in modalità di risparmio energetico
+    ja = 省電力モードで建物の 3D 表示がオフになっている
+    ko = 3D 빌딩은 절전 모드에서 꺼집니다.
+    mr = 3D इमारती वीज बचत मोडमध्ये बंद केल्या आहेत
+    nb = 3D-bygninger er slått av i strømsparingsmodus
+    nl = 3D-gebouwen worden uitgeschakeld in de energiebesparende modus
+    pl = Budynki 3D są wyłączone w trybie oszczędzania energii
+    pt = Edifícios 3D são desativados no modo de economia de energia
+    pt-BR = Edifícios 3D são desativados no modo de economia de energia
```

Comment:
```suggestion
    pt-BR = Os edifícios 3D são desligados em modo de economia de energia
```

Ideally, translations should be taken from DeepL, it has better results than Google in most cases. Also, Google doesn't have pt-BR, it always generates pt.

It would be great to leverage [DeepL API](https://www.deepl.com/docs-api) to translate to all its languages and add missing translations that are left using Google and the current script. Maybe a separate issue for that should be created.

Reply:
Updated `pt-BR` translation.

Do we have issue regarding translation API change? Maybe should keep Google Translate API as a backup?

Reply:
Actually Google Translate uses PT-BR, AFAIK.

For this string, I would use:

Os edifícios 3D são desativados no modo de economia de energia

Reply:
My bad, yes, Google always uses pt-BR, but they don't translate well into pt-PT.